internal/driver/glfw/window_generic.go

```diff
@@ -0,0 +1,530 @@
+package glfw
```

Comment:
And as above, can the core logic be left intact with platform specific functions where it differs?

Reply:
The problem is that the abstraction layer is also named glfw and isn't 1:1 compatible with the native glfw. It will also always be lagging behind. So I kind of choose to live with the more duplicated code and less work to do on upstream JS glfw. Still I agree, there is potential for more refactoring and better naming. Will take care of that.

Reply:
Check out internal/painter/gl where we solve the same problem